content/en/docs/security/config-best-practices.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Collector configuration best practices
+linkTitle: Collector config
+description: Best practices to securely configure OpenTelemetry Collector.
+weight: 20
+---
+
+When configuring the OpenTelemetry (OTel) Collector, consider the following
+practices to better secure your Collector instance.
+
+## Receivers and exporters
+
+We recommend enabling only the minimum required components for a Collector
+configuration: one receiver and one exporter. Configuring only the minimum set
+of components exposes a minimum set of required ports.
+
+- Receivers and exporters can be either push- or pull-based. In either case, the
+  connection established should be over a secure and authenticated channel.
+- Unused receivers and exporters should be disabled to minimize the attack
+  vector of the Collector.
+- Receivers and exporters might expose buffer, queue, payload, and worker
+  settings using configuration parameters. If these settings are available, you
+  should proceed with caution before modifying the default configuration values.
+  Improperly setting these values might expose the OpenTelemetry Collector to
+  additional attack vectors including resource exhaustion.
+
+### Safeguards against denial of service attacks
+
+Bind receivers' servers to addresses that limit connections to authorized users,
+so that your Collectors aren't exposed to the public internet or to wider
+networks than necessary.
+
+For example, if the OTLP receiver OTLP/gRPC server has only local clients, bind
+the `endpoint` setting to `localhost`:
+
+```yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: localhost:4317
```

Comment:
I don't recall the exact details, would this bind to both IPv4 and IPv6, or the Collector has a recommendation regarding how to specify IPv4/v6 explicitly?

Reply:
I've rewritten the DOS safeguard section as well, with updated information about `localhost`. 

Rather than provide an example of binding to a pod's IP, I linked to the [README section](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/security-best-practices.md#safeguards-against-denial-of-service-attacks). Unless you think that content (Docker, Kubernetes, etc.) is relevant to end users and belongs on the website? I was trying to draw a line between end user and developer content, but I'm happy to make changes.